### PR TITLE
Add BitNordic Gentx & S.Peer

### PR DESCRIPTION
**Pull request by wainach**
Add BitNordic Gentx & S.Peer for mainnet

### CODE REVIEW

**Review by YazzyYaz — APPROVED** [review]

File: celestia/peers.txt [hunk 1]
```diff
@@ -1 +0,0 @@
-
```
Comment (liamsi) [inline celestia/peers.txt:1]: Can you simply not modify this file please?

**Review by liamsi — CHANGES_REQUESTED** [review]

File: celestia/peers.txt (hunk 1 above)
Comment (wainach) [reply celestia/peers.txt:1]: Hey, I simply followed the announcement from Discord. But let me add it to the peers file instead.

Comment (liamsi) [reply celestia/peers.txt:1]: The discord announcement clearly states to create a separate file and not modify the existing celestia/peers.txt

**Review by liamsi — CHANGES_REQUESTED** [review]
https://github.com/celestiaorg/networks/pull/417#discussion_r1370617740

**Pull request closed without merging**